esmvalcore/config-developer.yml

```diff
@@ -194,3 +194,14 @@ CESM:
   output_file: '{project}_{dataset}_{case}_{gcomp}_{scomp}_{type}_{mip}_{short_name}'
   cmor_type: 'CMIP6'
   cmor_default_table_prefix: 'CMIP6_'
+
+ACCESS:
+  cmor_strict: false
+  input_dir:
+    default:
+      - '{institute}/{dataset_name}/{exp}/{modeling_realm}/netCDF'
+  input_file:
+    default: '{dataset}.{special_attr}-*.nc'
```

Comment:
Information like `special_attr` and `modeling_realm` can also go into the extra facets (see other comment) if they are always the same for a given variable. It would still be possible to overwrite them in the recipe, but they would not be mandatory facets anymore.